### PR TITLE
Add .about.yml test and Rake task to run it. Connects #37.

### PR DESCRIPTION
**Pull request by ertzeid**

### CODE REVIEW

**Comment by ccostino** [comment]
Looks like we need to investigate what's going on with Travis in this repo - I know it's not related to this PR, this happened on mine too (and that was just a text file change!).

File: .about.yml [hunk 1]
```diff
@@ -18,7 +18,8 @@ licenses:
     url: https://github.com/18F/team_api/blob/master/LICENSE.md
 tasks: https://github.com/18F/team_api/issues
 contact:
-- mailto:michael.bland@gsa.gov
+- url: mailto:michael.bland@gsa.gov
+  text: Michael Bland
```
Comment (ccostino) [inline .about.yml]: You should add yourself, you've contributed to this!

**Comment by ccostino** [comment]
Do I understand the flow of this correctly?

1. Include the team_api gem in your project.
2. Make sure your project has an .about.yml file.
3. Just run "rake run_about_yml_check" in your project root directory.
4. Check for errors in the output (which runs the test under the hood).

As a test scenario, I just ran the rake task in the repo itself locally and things appeared to work fine/as expected, so I just wanted to double check I was on the right track.

**Comment by ertzeid** [comment]
Using it in another repo will be slightly, but not much, more complex than
what you describe. Essentially, their Rakefile will have to load the rake
task file from the gem. I have the start of an update to do this for the
Hub, which I'll then use as an example for any other teams wanting to do
this, once it's an actual commit on master itself.

File: .about.yml (hunk 1 above)
Comment (ertzeid) [inline .about.yml]: Yes! But in a different commit. Will go add.

**Comment by ertzeid** [comment]
Moved this to the correct repo: https://github.com/18F/about_yml/pull/56
Closing this one...

**Pull request closed without merging**